app/web/features/search/LocationAutocomplete/LocationAutocomplete.stories.tsx

```diff
@@ -29,8 +29,8 @@ const Template: Story<any> = (args) => {
         params={urlParams.current}
       />
       <p>
-        Pressing enter in the field shouldn't perform a submit action, but the
-        button should.
+        Pressing enter in the field shouldn&apos;t perform a submit action, but
```

Comment:
What happens on the server side without converting the character to an escape sequence?

Reply:
Probably nothing but there was a new lint rule complaining. Maybe I should just disable it.

Reply:
I guess it doesn't matter that much as we should be moving these messages into JSON files too as part of the i18n work, so probably fine to leave it for now